src/foam/support/view/ReplyView.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.support.view',
+  name: 'ReplyView',
+  extends: 'foam.u2.View',
+
+  documentation: 'Internal note for ticket detail view',
+
+  requires: [
+    'foam.support.view.MessageCard'
+  ],
+
+  properties:[
+    {
+      name: 'variant',
+      postSet: function(oldValue, newValue){
+        this.viewData.variant = newValue;
+      }
+    },
+    {
+      class: 'String',
+      name: 'message',
+      view: 'foam.u2.tag.TextArea',
+      postSet: function(oldValue, newValue){
+        this.viewData.message = newValue;
+      }
+    }
+  ],
+
+  imports: [
+    'viewData'
+  ],
+  
+  exports: [
```

Comment:
If it wants to do this is should just remove the 'exports:' and just extend 'Controller' instead of 'View'.